forum/projects/permissions.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import permissions
+
+
+class UpdateOwnProject(permissions.BasePermission):
+    """
+    
+    """
+    def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS:
+            return True

```

Comment:
I think it is not good to use `request.user` if we use JWTs for login.
Try to retrieve the user_id of the current user from access_token.
As example, see the `ThisUserPermission` in users/permissions.py.